plugins/README.md

```diff
@@ -0,0 +1,28 @@
+## :plugins
+
+This folder holds all our Gradle- and Kotlin Compiler plugins:
+
+### [:plugins:dataframe-gradle-plugin](./dataframe-gradle-plugin)
+The Gradle plugin for DataFrame that can generate data schemas from a data sample using the simple
+task `dataframes {}`. It uses [:plugins:symbol-processor](./symbol-processor) to generate column accessors.
```

Comment:
> It uses [:plugins:symbol-processor](./symbol-processor) to generate column accessors.

It applies KSP plugin to your build and symbol-processor for easier configuration

Reply:
Oh, ok, exactly what you wrote in the next file :o